client/my-sites/customer-home/controller.jsx

```diff
@@ -21,3 +25,13 @@ export default function( context, next ) {
 
 	next();
 }
+
+export function maybeRedirect( context, next ) {
+	const state = context.store.getState();
+	const slug = getSelectedSiteSlug( state );
```

Comment:
I think it's redundant to declare `state` since you'll only use it once.I think it's best to do:

```js
const slug = getSelectedSiteSlug( context.store.getState() );
```